### PR TITLE
recommend using gpt-4o model

### PR DESCRIPTION
**Pull request by isidorn**

### CODE REVIEW

File: api/extension-guides/language-model.md [hunk 1]
```diff
@@ -192,9 +192,11 @@ We don't expect specific models to stay supported forever. When you reference a
 
 ### Choosing the appropriate model
 
-Extension authors can choose which model is the most appropriate for their extension. We recommend starting with less powerful models, such as `gpt-3.5-turbo`, because they are faster and might allow for a smoother user experience. You might use more powerful but slower models, such as  `gpt-4`, for complex tasks and only after the faster models prove to be inadequate.
-
-> **Note**: both `gpt-3.5-turbo` and `gpt-4` models have the limit of `4K` tokens. These limits will be expanded as we learn more how extensions are using the language models.
+Extension authors can choose which model is the most appropriate for their extension. We recommend using `gpt-4o` for it's performance and quality. `gpt-3.5-turbo`, and `gpt-4` are also available. To get a full list of available models, you can use this code snippet:
```
Comment (ntrogh) [inline api/extension-guides/language-model.md:195]:
```suggestion
Extension authors can choose which model is the most appropriate for their extension. We recommend using `gpt-4o` for its performance and quality. In addition, `gpt-3.5-turbo`, and `gpt-4` are also available. To get a full list of available models, you can use this code snippet:
```

File: api/extension-guides/language-model.md [hunk 2]
```diff
@@ -192,9 +192,11 @@ We don't expect specific models to stay supported forever. When you reference a
...
+```typescript
+const allModels = await vscode.lm.selectChatModels(MODEL_SELECTOR);
+```
+> **Note**: All models have the limit of `4K` tokens. The returned model object from `selectChatModels` call has the attribute `maxInputTokens` that shows the token limit. These limits will be expanded as we learn more how extensions are using the language models.
```
Comment (ntrogh) [inline api/extension-guides/language-model.md:199]:
```suggestion
> **Note**: All models have a limit of `4K` tokens. The returned model object from the `selectChatModels` call has a `maxInputTokens` attribute that shows the token limit. These limits will be expanded as we learn more about how extensions are using the language models.
```

**Review by ntrogh — CHANGES_REQUESTED** [review]
Just a few minor text edits, otherwise LGTM.

**Review by Tyriar — APPROVED** [review]

**Comment by isidorn** [comment]
@ntrogh you will also have to approve 👼

**Review by ntrogh — APPROVED** [review]
LGTM!

**Review by 1chooo — COMMENTED** [review]
That's great!

**Pull request merged**